SPECS-EXTENDED/cassandra/cassandra.spec

```diff
@@ -0,0 +1,209 @@
+%global __os_install_post %{nil}
+%global debug_package %{nil}
+
+Summary:        Cassandra is a highly scalable, eventually consistent, distributed, structured key-value store
+Name:           cassandra
+Version:        4.0.3
+Release:        1%{?dist}
+URL:            http://cassandra.apache.org/
+License:        Apache License, Version 2.0
+Group:          Applications/System
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        http://archive.apache.org/dist/cassandra/%{version}/apache-%{name}-%{version}-src.tar.gz
+Source1:        cassandra.service
+Source2:        cassandra-build-cache-4.0.3.tar.gz
```

Comment:
In addition to the `%prep` section, could you also mention that re-build instructions are in the readme file?

Reply:
Added comment in %prep to refer to Readme file for detailed instructions.